src/components/CountDownTimer.tsx

```diff
@@ -0,0 +1,108 @@
+import { CountdownCircleTimer } from "react-countdown-circle-timer";
+import "../styles.css";
+
+const minuteSeconds = 60;
+const hourSeconds = 3600;
+const daySeconds = 86400;
+
+const timerProps = {
+  isPlaying: true,
+  size: 120,
+  strokeWidth: 6
+};
+
+const renderTime = (dimension: any, time: number) => {
+  return (
+    <div className="time-wrapper">
+      <div className="time">{time}</div>
+      <div>{dimension}</div>
+    </div>
+  );
+};
+
+const getTimeSeconds = (time: number) => (minuteSeconds - time) | 0;
+const getTimeMinutes = (time: number) =>
+  ((time % hourSeconds) / minuteSeconds) | 0;
+const getTimeHours = (time: number) => ((time % daySeconds) / hourSeconds) | 0;
+const getTimeDays = (time: number) => (time / daySeconds) | 0;
```

Comment:
why are these bitwise 'or' with zero? I assume this is a hack to floor the numbers, however using Math.floor(num) is more explicit

Reply:
also, this doesn't really work for large numbers. e.g. 3000000000.1 | 0 = -1294967296, so this hack probably shouldn't be used